packages/shared/routes/dashboard/governance/views/GovernanceEventDetails.svelte

```diff
@@ -40,6 +60,17 @@
         }
     }
 
+    $: results = event?.status?.questions?.[0]?.answers?.filter(
+        (answer) => answer?.value !== 0 && answer?.value !== 255
+    )
+    $: totalVotesResult = results?.reduce((acc, val) => acc + val?.accumulated, 0)
+    $: displayedPercentages = results?.map((result) => {
```

Comment:
Maybe refactoring this in a separate function would bring more clarity in the code?